src/coreclr/tests/src/readytorun/coreroot_determinism/Program.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+internal class Program
+{
+    public static bool IsTimestampByte(int i)
+    {
+        return i >= 136 && i < 140;
+    }
+
+    public static int CompareDLLs(string folder1, string folder2)
+    {
+        foreach (string filepath1 in Directory.EnumerateFiles(folder1, "*.dll"))
+        {
+            byte[] file1 = File.ReadAllBytes(filepath1);
```

Comment:
It would be better to put all the diffing code into a try-catch block, and WriteLine the exception if any is thrown, so that we can get a chance to run the diff on all files instead of just failing on the first assembly with a non-deterministic output.